### DISCUSSION TITLE
Error while trying to install nuclei

### DISCUSSION THREAD

**Discussion by taku65maku** [projectdiscovery/nuclei#2111] [General]
![Screenshot from 2022-06-05 12-07-27](https://user-images.githubusercontent.com/48118772/172038690-c310e1e4-a333-4d6e-97d8-c1dde70dbe1f.png)

**Comment by taku65maku** [comment]
@xstevens

**Comment by taku65maku** [comment]
@ehsandeep @ehrishirajsharma @Marmelatze @revolunet

**Comment by ehsandeep** [comment]
@taku65maku please do avoid tagging individuals.

Looks like you have some permission issue on your system, instead go install or build you can try to download the compiled binary from here https://github.com/projectdiscovery/nuclei/releases, unzip the file and you have ready to run binary.

**Reply by taku65maku** [reply]
sorry sir this is my first time on discussion. sorry.

**Reply by ehsandeep** [reply]
No problem, let me know if you facing issues to install nuclei using release page.
